src/aat/resources/features/F-107/S-931.td.json

```diff
@@ -29,11 +29,11 @@
     }
   },
   "expectedResponse": {
-    "_extends_": "Common_400_Response",
+    "_extends_": "Common_422_Response",
     "body" : {
-      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException",
+      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.CaseValidationException",
```

Comment:
Now this is returning a normal `CaseValidationException`, we should also see the specific errors in the `details` field further down - no longer `null`.